tests/test_analysis.py

```diff
@@ -145,7 +145,11 @@ def test_can_analyze_c(self):
 class _NonSeekableEmptyBytesIO(BytesIO):
     # Class to create a 'dummy object that mimics a non-seekable file handle'
     def __init__(self):
-        self.seekable = False
+        super().__init__()
```

Comment:
Do we need the init at all, if there's nothing we do in it? Without it, the super-init should be called automatically.

If for some reason the init remains, at least remove the commented out code below.

Reply:
Removed the `__init__` method and it works fine: [0888a5e](https://github.com/roskakori/pygount/pull/115/commits/0888a5e0dba4b1d58a64f5c9073a51be1a337669)